cmd/check/root_test.go

```diff
@@ -14,6 +14,8 @@ func TestCheckCommand(t *testing.T) {
 	ts := client.NewTestServer(t, client.ReadServer, []*namespace.Namespace{nspace}, newCheckCmd)
 	defer ts.Shutdown(t)
 
-	stdOut := ts.Cmd.ExecNoErr(t, "subject", "access", nspace.Name, "object")
+	stdOut := ts.Cmd.ExecNoErr(t, "subject", "access", nspace.Name, "object",
+		"--insecure-skip-hostname-verification=true",
```

Comment:
I think this should be enough
```suggestion
		"--insecure-skip-hostname-verification",
```

Reply:
Yea I thought so too, but something broke in the flag parsing then, it tried to swallow the next param.